ecs-logs-collector.sh

```diff
@@ -426,6 +429,9 @@ get_system_services() {
     amazon)
       chkconfig --list > ${info_system}/services.txt 2>&1
       ;;
+    amazon2)
+      chkconfig --list > ${info_system}/services.txt 2>&1
```

Comment:
AL2 is systemd based, so we should gather the services with `systemctl`, like so:

```suggestion
      systemctl list-units > ${info_system}/services.txt 2>&1
```

Reply:
Agree.